plugins/modules/aci_contract_subject.py

```diff
@@ -26,6 +26,12 @@
     - The contract subject name.
     type: str
     aliases: [ contract_subject, name, subject_name ]
+  direction:
```

Comment:
Usage of variable names similar to that on UI is recomended (direction -> apply_both_direction)

Reply:
ok